feather/server/src/options.rs

```diff
@@ -28,6 +28,12 @@ pub struct Options {
     /// The default gamemode for new players.
     pub default_gamemode: Gamemode,
 
+    /// The default tablist header.
+    pub tablist_header: Text,
+
+    /// The default tablist footer.
+    pub tablist_footer: Text,
+
```

Comment:
We have `Config` for server config and `Options` for `Server` options (mostly for initial connection handling, status, ping, working with proxies), so I think tablist values should only be in `Config`.

Reply:
```
let server_options = game.resources.get::<Server>().unwrap().options.clone();

game.insert_resource(TablistHeaderFooter {
    header: server_options.tablist_header.clone(),
    footer: server_options.tablist_footer.clone(),
});
```

Then how do I get it from config to use it for `TablistHeaderFooter` in tablist.rs.

Reply:
Nevermind, got it.

Reply:
You can pass it to register(game) function, or even better, make `Config` (or even `Rc<Config>` to make it immutable) a resource and `Options` non-resource. `Options` is never used as a resource (just did a quick search on cs.github)